wrapper_test.go

```diff
@@ -351,11 +354,40 @@ func (w *wrapperTestSuite) TestCheckFailWriteSpanHandler() {
 	handler := reflect.ValueOf(lambdaHandler)
 	_ = handler.Call([]reflect.Value{reflect.ValueOf(context.Background()), reflect.ValueOf(inputPayload)})
 
-	dirEntries, err = os.ReadDir(SPANS_DIR)
+	dirEntries, err := os.ReadDir(SPANS_DIR)
 	assert.NoError(w.T(), err)
 	assert.Equal(w.T(), 2, len(dirEntries))
+	var startSpanFilename, endSpanFilename string
 	for _, dirEntry := range dirEntries {
-		assert.NotEqual(w.T(), "balagan_end", dirEntry.Name())
+		if strings.Contains(dirEntry.Name(), "span") {
+			startSpanFilename = dirEntry.Name()
+		} else if strings.Contains(dirEntry.Name(), "end") {
+			endSpanFilename = dirEntry.Name()
+		}
+		assert.NotEqual(w.T(), "balagan_stop", dirEntry.Name())
 	}
+	assert.NotEmpty(w.T(), startSpanFilename)
+	assert.NotEmpty(w.T(), endSpanFilename)
+
+	fileBytes, err := ioutil.ReadFile(path.Join(SPANS_DIR, startSpanFilename))
+	assert.NoError(w.T(), err)
+	assert.Equal(w.T(), 3764, len(fileBytes))
```

Comment:
I think that the number `3764` will break on each change that we'll do to the data in the span.
Maybe it's better to do something like:
```suggestion
	assert.NotEmpty(w.T(), fileBytes)
```